app/assets/stylesheets/components/_slabs.scss

```diff
@@ -37,6 +37,10 @@
   background-color: $color-grey-light;
 }
 
+.slab--green {
+  background-color: #E8FDF7;
```

Comment:
seems like we're using this in a couple places now -- wondering if we should turn this into a variable?